server/utils/cloudFunctions/functions/firebaseConfig.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Description: Initialize the index environment by 
+ *              initializng the firebase api to access the dev/prod ads db
+ * Author: Robert Marcus
+ * Date: July 7, 2020
+ */
+
+// Initialize the firebase functions sdk. 
+const functions = require('firebase-functions');
```

Comment:
nit: please make this more clear - something like `firebaseFunctions` would make more sense, so you don't need to go back to this file to see what `functions` actually mean